connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -623,6 +625,16 @@ class SparkConnectPlanner(session: SparkSession) {
         val children = fun.getArgumentsList.asScala.toSeq.map(transformExpression)
         Some(In(children.head, children.tail))
 
+      case "nth_value" if fun.getArgumentsCount == 3 =>
```

Comment:
so sad, not worth fixing?

Reply:
if we fix it by add a new constructor in `NthValue`, then I guess it will also change the sql syntax:

before:
`SELECT nth_value(four, 0) OVER (ORDER BY ten), ten, four FROM tenk1;`

after:
`SELECT nth_value(four, 0) OVER (ORDER BY ten), ten, four FROM tenk1;`
`SELECT nth_value(four, 0, TRUE) OVER (ORDER BY ten), ten, four FROM tenk1;` // new parameter